### ISSUE TITLE
Navigation Drawer in MainActivity for categories like food, places etc.

### ISSUE THREAD

**Issue by Rondaul** [UdacityAndroidDevScholarship/explore-india#22]

**Comment by sidhuparas** [comment]
Please provide more description. How are you planning to implement it. Also, a visual mockup would be good. Do share your mockup in #cp_explore_india by today.

**Comment by Rondaul** [comment]
It will take some time for mockup and I don't know if its too soon. But I was thinking to have categories like in maps something like this : 
![androidpit-google-maps-update-offline-areas-w782](https://user-images.githubusercontent.com/14847906/38767591-dcbb60fe-4001-11e8-9871-c960bd5d0701.png)


But , instead of terrains and traffics. We shall have category for food, interesting place and culture at different locations etc in the nav drawer and when one clicks on the category, the specific category will only be shown on the map. What do you think about it? @sidhuparas

**Comment by tavishjain** [comment]
@Rondaul do we have to create a new branch for the code we are updating ?
As the readme says : "Name the branch something like fixes-xxx-issue or add-xxx-feature where xxx is a short description of the changes or feature you are adding."

**Comment by Rondaul** [comment]
@tavishjain  yes for every issue you are trying to work on first create a new branch, work on it. merge it in you own master branch first, check it if it works fine. Then you can push it to you github repo and create a pull request from there. Once the pull request is accepted, you can delete that branch.

**Comment by theshivamlko** [comment]
Currently there will be No Food catgeories
Only  Places
